### ISSUE TITLE
[BUG] Zoomed in Image close button has color contrast issues over some images

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5429]
**Describe the bug**
When the user has zoomed in on just the image from an Event or Group, there is a close button "X" on the screen. This is white. The arrow that appears is black. They should be consistent and meeting color contrast ratio minimum. 
[WCAG 1.4.3 ](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**To Reproduce**
1. Go to "Groups" or "Events"
2. Click on a group you have created or are the admin of
3. Click the hamburger menu for options
4. Select "Group admin settings"
5. Click "Upload Cover Image" 

**Expected behavior**
Vanessa shared this image in which the arrow is below the image and only the image and "X" appear:

<img width="1206" height="2622" alt="Image" src="https://github.com/user-attachments/assets/f01693d9-99b1-4785-bd5f-cb253bdeb228" />

This matches with the screen reader behavior shown in screenshots section. 

**Actual behavior**
I used a larger image and there was both a Back arrow and the "X" close button. Please see video for how the TalkBack interacted with the image zoomed in. 

**Screenshots**

https://github.com/user-attachments/assets/4afc89e3-356e-49a9-91af-6e8721e1ed92

**Smartphone (please complete the following information):**
• OS: Android Version 15
• Device: Google Pixel 8a
APK Version 7.3.8

**Additional context**
 (I Uploaded a large screenshot with poor contrast color intentionally to test my Android Accessibility scanner for Color contrast. The tool did not find it. However, I used Apowermirror software to mirror the Android to my desktop then used Color contrast analyzer tool and found the "X" over my image to fail. This is an edge case but we still want to aim for color continuity and behavior for the Back arrow and the X. Per Vanessa, she agrees we do not need both, or know why the back arrow appeared here. We can just have the "X" close button and ensure that it can be reached by screen reader.)

**Comment by TodorBachvarov** [comment]
@acovher2 I've increased the alpha of the border color from 0.3(30%) to 0.7(70%) in version 7.3.29
